content/articles/creating-dummy-data-in-python-using-faker-package/index.md

```diff
@@ -0,0 +1,105 @@
+Sound finish set draw notice imagine that. Blue between least democratic down week wait. Reduce inside me.
+```
+#### 4. Creating unique dummy data
+You can utilize the generator's.unique property to ensure that the dummy data created is unique.
+```python
+# generates 10 unique texts 
+texts = [ourFake.unique.name() for b in range(10)]
+```
+#### 5. Command-line usage of Faker package
+Command-line invocation of the faker package allows you to directly type in the command prompt code.
+
+#### 6. Creating localized dummy data
+The Faker Generator may generate localized false data if a place is provided as an input. Moreover, it can be translated into a wide range of languages. Otherwise, all locales must use the python list data type. 
+
+The following are some names we can come up with:
+```Python
+ourFake = Faker()
+for b in range(10):
+    print(ourFake.name())
+```
+Output:
+```bash
+Daniel Davidson
+Kristin Stewart
+Derrick Tran
+Matthew Mccarty
+Kevin Davis
+Kim Watkins
+Ashley Humphrey
+Corey Webb
+Melissa Barrera
+Juan Greene
+```
+### Other dummy data creation methods
+They are as follows:
+
+1. In Python, Numpy's Random module can be accessed: Pseudorandom numbers can be generated with the random package. Many functions are available, including `rand()`, `randint()`, and many more.
+**WExample**
+```python
+import numpy as num
+myArray = num.random.rand(5)
+print("Array : \n", myArray);
+```
+Output:
+```bash
+Array : 
+ [0.02471149 0.41561035 0.76783821 0.89628689 0.8540258 ]
+```
+2. Fauxfactory: Automated testing may be made easier with FauxFactory's random data generator!. When building tests for your application, you may need to provide the sections you're testing with random, non-specific data. To test your code quickly, you can use this anytime. [Here](https://fauxfactory.readthedocs.io/en/latest/#), you may learn more about it.
+###  A few more words regarding dummy data
+As a result of dummy variables, a single regression equation like y= a + bc might represent multiple groups. The term dummy variable refers to a numerical variable that is used to represent categorical information. The necessity for different equations for each subgroup is eliminated. Putting it in another way, dummy variables are like switches in an equation that can enable or disable other variables.
+#### Effect of a dummy variable
+When the coefficient of an explanatory or independent variable is set to zero, it cannot affect the dependent variable. Still, the intercept will be affected when its coefficient is set to 1.
+
+Dummy variables are created via one-hot encoding. One-hot Encoding transforms categorical variables into a form that can be fed into machine learning algorithms to help them perform better. One of these variables must be used as the starting point to avoid perfect multicollinearity. Multicollinearity occurs when the correlations between two or more independent variables are extremely high in a regression model.

```

Comment:
Add examples of one hot encoding.

Reply:
`One of these variables must be used as the starting point to avoid perfect multicollinearity. Multicollinearity occurs when the correlations between two or more independent variables are extremely high in a regression model.`

Please be more brief about this